tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraEmailSendMethodContract.scala

```diff
@@ -55,154 +78,1613 @@ trait LinagoraEmailSendMethodContract {
+    s"""
+       |{
+       |  "using": [
+       |    "urn:ietf:params:jmap:core",
+       |    "urn:ietf:params:jmap:mail",
+       |    "urn:ietf:params:jmap:submission",
+       |    "com:linagora:params:jmap:pgp"
+       |  ],
+       |  "methodCalls": [
+       |    [
+       |      "Email/send",
+       |      {
+       |        "accountId": "$ACCOUNT_ID",
+       |        "create": {
+       |          "K87": {
+       |            "email/create": {
+       |              "mailboxIds": {
+       |                "${getBobInboxId(server).serialize}": true
+       |              },
+       |              "subject": "World domination",
+       |              "htmlBody": [
+       |                {
+       |                  "partId": "a49d",
+       |                  "type": "text/html"
+       |                }
+       |              ],
+       |              "bodyValues": {
+       |                "a49d": {
+       |                  "value": "$HTML_BODY",
+       |                  "isTruncated": false,
+       |                  "isEncodingProblem": false
+       |                }
+       |              }
+       |            },
+       |            "emailSubmission/set": {
+       |              "envelope": {
+       |                "mailFrom": {
+       |                  "email": "${BOB.asString}"
+       |                },
+       |                "rcptTo": [
+       |                  {
+       |                    "email": "${ANDRE.asString}"
+       |                  }
+       |                ]
+       |              }
+       |            }
+       |          }
+       |        }
+       |      },
+       |      "c1"
+       |    ]
+       |  ]
+       |}""".stripMargin
+
+  private def getBobInboxId(server: GuiceJamesServer): MailboxId =
+    server.getProbe(classOf[MailboxProbeImpl])
+      .getMailboxId(MailboxConstants.USER_NAMESPACE, BOB.asString, MailboxConstants.INBOX)
+
   @Test
   def emailSendShouldReturnSuccess(server: GuiceJamesServer): Unit = {
+    server.getProbe(classOf[MailboxProbeImpl]).createMailbox(MailboxPath.inbox(ANDRE))
+
+    val response: String = `given`
+      .body(generateRequestBobSendMailToAndre(server))
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
 
+    assertThatJson(response)
+      .isEqualTo(
+        s"""
+           |{
+           |    "sessionState": "${SESSION_STATE.value}",
+           |    "methodResponses": [
+           |        [
+           |            "Email/send",
+           |            {
+           |                "accountId": "$ACCOUNT_ID",
+           |                "newState": "$${json-unit.ignore}",
+           |                "created": [
+           |                    [
+           |                        "K87",
+           |                        {
+           |                            "emailSendId": "$${json-unit.ignore}",
```

Comment:
What does this correspond to?

Reply:
We can use it in the future when need `recordCreationIdInProcessingContext`. IMO 

I will remove it